src/software/backend/input/network/networking/proto_multicast_listener.h

```diff
@@ -41,15 +42,19 @@ class SSLGameControllerClient
     void handleDataReception(const boost::system::error_code& error,
                              size_t num_bytes_received);
 
-    // A UDP socket that we listen on for protobuf messages from SSL Game Controller
+    // A UDP socket that we listen on for ReceiveProto messages from the network
     boost::asio::ip::udp::socket socket_;
-    // The endpoint for the sender, in this case SSL Game Controller
+    // The endpoint for the sender
     boost::asio::ip::udp::endpoint sender_endpoint_;
 
     // The maximum length of the buffer we use to receive data packets from the network
     static constexpr unsigned int max_buffer_length = 4096;
     // Acts as a buffer to store the raw received data from the network
     std::array<char, max_buffer_length> raw_received_data_;
-    // The function to call on every received packet of vision data
-    std::function<void(Referee)> handle_function;
+    // The function to call on every received packet of ReceiveProto data
+    std::function<void(ReceiveProto)> handle_function;
```

Comment:
super uber duper nit: "handle" makes me think the processing is happening in the class. I think `receive_callback` or similar is slightly more clear it's an external callback function.

Reply:
done